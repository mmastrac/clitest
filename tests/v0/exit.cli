#!/usr/bin/env clitest --v0

exit script;

$ echo "won't run!"
