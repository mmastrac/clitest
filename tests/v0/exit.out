Running <root>/exit.cli ...

exiting script

<root>/exit.cli PASSED
